standalone-examples/src/main/java/io/pravega/example/iot/TurbineHeatSensor.java

```diff
@@ -100,8 +99,7 @@ public static void main(String[] args) throws Exception {
                     .scalingPolicy(policy)
                     .build();
             streamManager.createStream(scopeName, streamName, config);
-        }
-        catch (URISyntaxException e) {
+        } catch (URISyntaxException e) {
```

Comment:
It is great to fix format changes, but let's not interleave them in PRs like this. It makes the PR longer and harder to differentiate a change that is related to the issue that is being resolved.

Reply:
Reverted this change.